lunatrace/bsl/generated-lunatrace-tmuxp.yaml

```diff
@@ -0,0 +1,42 @@
+session_name: lunatrace
+windows:
+  - window_name: services
+    layout: tiled
+    panes:
+      - shell_command:
+          - cd ory/oathkeeper
+          - oathkeeper --config config.yaml serve
+      - shell_command:
+          - cd backend
+          - >-
+            GITHUB_APP_ID="179126" GITHUB_APP_PRIVATE_KEY="$(cat
+            ../github-app-dev.2022-03-09.private-key.pem | base64 -w0)" yarn run
+            start:server
+      - shell_command:
+          - cd hasura
+          - >-
+            /home/breadchris/projects/lunasec-monorepo/tools/service-scripts/wait-for-it.sh
+            -h localhost -p 8080 && sleep 3
+          - hasura console
+      - shell_command:
+          - cd frontend
+          - yarn run start:server
+      - shell_command:
+          - sudo docker-compose down && sudo docker-compose up
+  - window_name: etl
+    layout: tiled
+    panes:
+      - shell_command:
+          - cd backend
+          - >-
+            GITHUB_APP_ID="179126" GITHUB_APP_PRIVATE_KEY="$(cat
+            ../github-app-dev.2022-03-09.private-key.pem | base64 -w0)"
+            QUEUE_NAME="lunatrace-EtlStorage-ProcessManifestProcessingQueue7A1F1CB2-wVcNqaEtNUJP"
+            QUEUE_HANDLER="process-manifest-queue" yarn run start:etl
```

Comment:
Im not even sure its fair to call what the workers do "ETL" and there will be more soon that do other things even less related to ETL.  Id probably call it "worker"